Source/WTF/wtf/StackTrace.cpp

```diff
@@ -45,6 +45,55 @@ void WTFGetBacktrace(void** stack, int* size)
 
 namespace WTF {
 
+#if USE(LIBBACKTRACE)
+static struct backtrace_state* backtraceState()
+{
+    static struct backtrace_state* backtraceState = backtrace_create_state(nullptr, 1, nullptr, nullptr);
+    return backtraceState;
+}
+
+static void backtraceSyminfoCallback(void* data, uintptr_t, const char* symname, uintptr_t, uintptr_t)
+{
+    const char** symbol = static_cast<const char**>(data);
+    *symbol = symname;
+}
+
+static int backtraceFullCallback(void* data, uintptr_t, const char*, int, const char* function)
+{
+    const char** symbol = static_cast<const char**>(data);
+    *symbol = function;
+    return 0;
+}
+
+const char** symbolize(void* const* addresses, int size)
+{
+    struct backtrace_state* state = backtraceState();
+    if (!state)
+        return nullptr;
+
+    const char** symbols = (const char**)malloc(sizeof(const char*) * size);
```

Comment:
Style: use `static_cast` wherever possible and `reinterpret_cast` otherwise; only do a C-style cast when the type you're casting from is uncertain and could be either a pointer or an integer, which is basically just EGL types.

```suggestion
    const char** symbols = static_cast<const char**>(malloc(sizeof(const char*) * size));
```

Reply:
Done.